playbooks/operationalizing/add_node.adoc

```diff
@@ -0,0 +1,120 @@
+---
+---
+= OpenShift Enterprise 3 Adding nodes - "ScaleUP"
+Øystein Bedin <bedin@redhat.com>
+v1.0, 02-Feb-2016
+:scripts_repo: https://github.com/rhtconsulting/rhc-ose
+:toc: macro
+:toc-title:
+
+toc::[]
+
+After running OpenShift for awhile, there may be a need to add in additional capacity for hosting applications and pods. This can be done by adding additional "compute nodes", or OpenShift nodes, to the existing infrastructure without having to go through a re-install. One option is to re-run the installation with an updated inventory file, and additional nodes. However, this can be a bit risky as it will revert custom configurations made after initial installation back to "fresh install" state. A better mechanism is to use the "scaleup" playbook part of the "openshift-ansible" installation implementation. This document describes the steps necessary to do this work and gotcha's that you may have to consider. 
+
+
+== Prerequisites
+
+The following prerequisites are required prior to beginning the steps to "ScaleUP" the environment 
+
+* A healthy and operational OpenShift Environment
+** Version 3.1.1 is recommended, but earlier 3.1.x versions may work as well after updating the "openshift-ansible" package
+* The current "inventory" file used for install (or any previous scaleup runs) of the OpenShift Environment
```

Comment:
This [inventory](http://docs.ansible.com/ansible/intro_inventory.html) link might be helpful to include